cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/app/etl/realtime/ETLWorker.java

```diff
@@ -183,15 +184,15 @@ private void initializeSource(WorkerContext context) throws Exception {
 
   @SuppressWarnings("unchecked")
   private void initializeSinks(WorkerContext context) throws Exception {
-    List<String> sinkPluginIds = GSON.fromJson(context.getSpecification().getProperty(Constants.Sink.PLUGINIDS),
-                                               STRING_LIST_TYPE);
+    List<SinkInfo> sinkPluginIds = GSON.fromJson(context.getSpecification().getProperty(Constants.Sink.PLUGINIDS),
```

Comment:
how come this is being changed? workers only need the ids right?  And the property seems like it should still be an list of ids (it's set in configure which hasn't changed)